src/fal/fal_script.py

```diff
@@ -53,30 +75,55 @@ def __init__(self, config: RuntimeConfig):
 class Context:
     current_model: Union[CurrentModel, None]
     config: ContextConfig
+    _arguments: Optional[Dict[str, Any]] = field(repr=False, default=None)
+
+    @property
+    def arguments(self) -> Dict[str, Any]:
+        if self._arguments is None:
+            raise ValueError(
+                "'context.arguments' is only accessible from hooks, "
+                "not from scripts/models"
```

Comment:
Why do you think is valuable to make this distinction?

Reply:
> Why do you think is valuable to make this distinction?

Not a distinction per se, but this is specifying the remaining types of code that can access `context` (`pre-hook`/`post-hook` can access it, `scripts` and `python models` can't.)